eva-server/src/main/java/uk/ac/ebi/eva/server/ws/AnnotationMetadataWSServerV2.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * European Variation Archive (EVA) - Open-access database of all types of genetic
+ * variation data from all species
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.server.ws;
+
+import io.swagger.annotations.Api;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+import uk.ac.ebi.eva.commons.core.models.AnnotationMetadata;
+import uk.ac.ebi.eva.commons.mongodb.services.AnnotationMetadataService;
+import uk.ac.ebi.eva.lib.eva_utils.DBAdaptorConnector;
+import uk.ac.ebi.eva.lib.eva_utils.MultiMongoDbFactory;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/v2/annotation-metadata", produces = "application/json")
```

Comment:
Should we make this URL `/v2/annotation-versions` to make its purpose more obvious? Or even open the door to show info about a species or assembly with `/v2/species/{species}/assembly/{assembly}/annotations`? The latter looks cool but we may prefer that for a hypothetical v3, when everything is more stable.

Reply:
in the future annotation-metadata could contain other info than the versions, although I can't think of anything to put there, so I'm fine with /v2/annotation-versions.

I'm not sure I like `/v2/species/{species}/assembl*ies*/{assembly}/annotations`, as it kind of suggests that species are containers of assemblies, which doesn't sound elegant to me.

Reply:
I have changed this to `/v2/annotation-versions` for now :)